common/src/types/http/foods/food-data.ts

```diff
@@ -0,0 +1,47 @@
+export type FoodDataEntryResponse = FoodLocalResponse & FoodAttributeResponse & FoodAdditionalDataResponse & CaloriesPer100gResponse
+
+export type FoodLocalResponse = {
+	code: string;
+	localDescription: string;
+	portionSizeMethods: PortionSizeMethodsResponse[] | [];
```

Comment:
does this provide any type advantage over just `PortionSizeMethodsResponse[]`? <-- this allows empty array as well.

Reply:
yeah, it is actually redundant at that point, nice spotting it!